spec/classes/python_spec.rb

```diff
@@ -272,4 +272,75 @@
       end
     end
   end
+
+  context "on a Gentoo OS" do
+    let :facts do
+      {
+        :id => 'root',
+        :kernel => 'Linux',
+        :osfamily => 'Gentoo',
+        :operatingsystem => 'Gentoo',
```

Comment:
you must specify `:lsbdistcodename` here for TravisCI to be happy.

https://travis-ci.org/stankevich/puppet-python/jobs/166102244